src/packer.pkr.hcl

```diff
@@ -75,6 +75,7 @@ data "amazon-ami" "windows_server_2022" {
   region      = var.build_region
 }
 
+
```

Comment:
I don't think there is any need for this blank line:
```suggestion
```

Reply:
@jsf9k committed!